edgepi-thermocouple.js

```diff
@@ -33,10 +33,13 @@ module.exports = function (RED) {
         try {
           output = msg.payload || output;
           let temps = await tc.singleSample();
-          if (output === "cj-lin") {
+          // if (output === "cj-lin") {
+          //   msg.payload = temps;
+          // } else {
+          //   msg.payload = output === "cj" ? temps[0] : temps[1];
+          // }
+          if (output === "lin") {
             msg.payload = temps;
```

Comment:
I think the payload should be `temps[1]`. check the sdk or test it.

Reply:
updated